### ISSUE TITLE
Oracle exception ORA-3113 during random row validation

### ISSUE THREAD

**Issue by nj1973** [GoogleCloudPlatform/professional-services-data-validator#1213]
Sample command reported to be raising ORA-3113:
```
data-validation validate custom-query row --source-conn ora --target-conn pg \
-sq "select * from dvt_test.t1 where ID between 100 and 500" \
-tq "select * from dvt_test.t1 where id between 100 and 500"  \
--primary-keys id  --hash id  \
--use-random-row  --random-row-batch-size 5
```

Exception:
```
    cursor.execute(statement, parameters)
cx_Oracle.DatabaseError: DPI-1080: connection was closed by ORA-3113
```

**Comment by helensilva14** [comment]
@nj1973 could you please check if this exception still happens? Regardless, I'll change its label and added low priority for now

**Comment by nj1973** [comment]
I was never able to reproduce it and was waiting on more information from the customer. I'll close it because we have nothing useful recorded here.
